lib/rubygems/remote_fetcher.rb

```diff
@@ -21,17 +22,13 @@ class Gem::RemoteFetcher
   class FetchError < Gem::Exception
     ##
     # The URI which was being accessed when the exception happened.
-
```

Comment:
The blank line may have been removed unintentionally.

Reply:
Done [`6f39f98` (#4868)](https://github.com/rubygems/rubygems/pull/4868/commits/6f39f9802d374700869f836e34a3570db8e9d0a8)